lib/model/folder_summary.go

```diff
@@ -76,8 +76,56 @@ func (c *folderSummaryService) String() string {
 	return fmt.Sprintf("FolderSummaryService@%p", c)
 }
 
-func (c *folderSummaryService) Summary(folder string) (map[string]interface{}, error) {
-	var res = make(map[string]interface{})
+type FolderSummary struct {
+	Errors     int
```

Comment:
JSON tags for keeping same casing as previously?

Reply:
TIL (again probably): Lowercasing is not the default behaviour for the json marshaller.